Heroku error [right now](https://dashboard.heroku.com/apps/af83f4b1-a56d-4e9a-9d70-5f084337c181/activity/builds/754982b7-a391-40f6-b4d0-1dc65a09b943):
```
-----> Installing dependencies
       Installing node modules (package.json)
       npm ERR! code ECONNRESET
       npm ERR! errno ECONNRESET
       npm ERR! network Invalid response body while trying to fetch https://registry.npmjs.org/@prisma%2fget-platform: aborted
       npm ERR! network This is a problem related to network connectivity.
       npm ERR! network In most cases you are behind a proxy or have bad network settings.
       npm ERR! network 
       npm ERR! network If you are behind a proxy, please make sure that the
       npm ERR! network 'proxy' config is set properly.  See: 'npm help config'
       
       npm ERR! A complete log of this run can be found in: /tmp/npmcache.oRYfO/_logs/2024-02-08T11_22_45_904Z-debug-0.log
-----> Build failed
       
       We're sorry this build is failing! You can troubleshoot common issues here:
       https://devcenter.heroku.com/articles/troubleshooting-node-deploys
       
       Some possible problems:
       
       - Node version not specified in package.json
         https://devcenter.heroku.com/articles/nodejs-support#specifying-a-node-js-version
       
       - ECONNRESET issues may be related to npm versions
         https://github.com/npm/registry/issues/10#issuecomment-217141066
       
       Love,
       Heroku
       
 !     Push rejected, failed to compile Node.js app.
 !     Push failed
 ```

--- a/.github/scripts/check-engines-client.sh
+++ b/.github/scripts/check-engines-client.sh
@@ -151,8 +151,7 @@ elif [ -f "$qe_location" ]; then
   echo "✔ Correct Query Engine exists at ${qe_location}"
 else
   echo "❌ Could not find Query Engine in ${qe_location} when using ${os_name}"
+  echo "$ ls $GENERATED_CLIENT"
+  ls $GENERATED_CLIENT
   exit 1
 fi
-
-echo "$ ls $GENERATED_CLIENT"
-ls $GENERATED_CLIENT

--- a/.github/scripts/check-engines-client.sh
+++ b/.github/scripts/check-engines-client.sh
@@ -125,13 +125,13 @@ elif [ $CLIENT_ENGINE_TYPE == "wasm" ]; then
   echo "WasmEngine: Enabled"
   case $os_name in
     linux)
-      qe_location="$GENERATED_CLIENT/query-engine.wasm"
+      qe_location="$GENERATED_CLIENT/query_engine_bg.wasm"
       ;;
     osx)
-      qe_location="$GENERATED_CLIENT/query-engine.wasm"
+      qe_location="$GENERATED_CLIENT/query_engine_bg.wasm"
       ;;
     windows*)
-      qe_location="$GENERATED_CLIENT\query-engine.wasm"
+      qe_location="$GENERATED_CLIENT\query_engine_bg.wasm"
       ;;
     *)
       os_name=notset

